themes/aframe/layout/partials/examples/iframe.ejs

```diff
@@ -2,4 +2,8 @@
         width="100%"
         height="100%"
         allowfullscreen="yes" scrolling="no"></iframe>
-  <a href="#" class="btn example__viewsource" id="exampleViewsource" target="_blank">View Source</a>
+
+  <div class="example__controls">
+    <a href="" class="btn" rel="nofollow">Replay</a>
+    <a href="#" class="btn" id="exampleViewsource" target="_blank">View Source</a>
```

Comment:
actually, instead of injecting this all with JS, we could do `href="<%- page.source_url %>"`. (and I'd probably also give this a `class` of `exampleViewsource` too.)

in a half-done rewrite to make this a SPA, I overcomplicated this with JS - my bad